msteams-platform/tabs/how-to/authentication/auth-silent-aad.md

```diff
@@ -112,3 +112,14 @@ if (authContext.isCallback(window.location.hash)) {
     }
 }
 ```
+
+### Handle sign out flow
+
+Use the following code to handle sign out flow in AAD Auth:
```

Comment:
Please give some insight on sign out flow, "While we do logout for teams tab or Bot, we clear the current session"